packages/embark/src/lib/utils/utils.js

```diff
@@ -568,7 +568,11 @@ function jsonFunctionReplacer(_key, value) {
 
 function getWindowSize() {
   const windowSize = require('window-size');
-  return windowSize.get();
+  if (windowSize) {
+    return windowSize.get();
+  }
+
+  return {width: 240, height: 75};
```

Comment:
This is a half way fix so I can debug on VSCode. It so happens that the console will take keyboard input, which allowed me to trigger the debugger where I needed it.